README.md

```diff
@@ -22,7 +22,7 @@ npm install
 2. Run local ethereum node, e.g. `ganache`. Make sure to use version greater than 6.12.2.
 
 ```bash
-npx ganache-cli --port 7545 --mnemonic "amused glory pen avocado toilet dragon entry kitchen cliff retreat canyon danger"
+npx ganache-cli --port 8545 --mnemonic "amused glory pen avocado toilet dragon entry kitchen cliff retreat canyon danger"
```

Comment:
problem in that. https://docs.openzeppelin.com/test-helpers/0.5/configuration#provider
I can set it to 8545 or i need to change this in code as a configuration

Reply:
or we can switch to test environment https://docs.openzeppelin.com/test-environment/0.1/

Reply:
``` 
In a Truffle environment, the web3 provider will be pulled from Truffle’s global web3 instance. 
Otherwise, this value defaults to http://localhost:8545.
```

I think it means it would get it from what is set here: `https://github.com/mysteriumnetwork/payments-smart-contracts/blob/master/truffle-config.js`. 

But anyway, this is tiny problem, so we can go forward with 8545 for now.